java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -85,10 +105,19 @@ public HttpConsumer(HttpConsumerConfig config, CountDownLatch messagesReceivedLa
     }
 
     public void createConsumer() throws IOException, InterruptedException, URISyntaxException {
-        String consumerInfo = "{\"name\":\"" + this.config.getClientId() + "\",\"format\":\"json\",\"auto.offset.reset\":\"earliest\"}";
+        Properties props = config.getProperties();
+        enableAutoCommit = Boolean.parseBoolean(props.getProperty(ENABLE_AUTO_COMMIT_CONFIG));
```

Comment:
I would set this in the HttpConsumerConfig after properties.putAll line in the fromEnv() method, adding enableAuthCommit as a field to HttpConsumerConfig class

Reply:
Done